core/src/main/java/org/apache/brooklyn/util/core/config/ConfigBag.java

```diff
@@ -367,6 +367,22 @@ public Object getStringKey(String key) {
         }
     }
 
+    /** Puts into this bag the raw value for the given key in the given bag, if it was present. */
+    @Beta
+    public <T> ConfigBag copyRaw(ConfigBag source, ConfigKey<T> key) {
```

Comment:
or in line with the suggestion on `configureCopy...` below

    copyKey(ConfigBag, ConfigKey<?>)
    copyKeys(ConfigBag, ConfigKey<?>, ConfigKey<?>, ...ConfigKey<?>)
    copyKeyAs(ConfigBag, ConfigKey<?> sourceName, ConfigKey<?> targetName)